docs/source/class_basics.rst

```diff
@@ -77,6 +77,53 @@ to it explicitly using ``self``:
            a = self
            a.x = 1      # Error: 'x' not defined
 
+Class and instance variable annotations
+***************************************
+
+Mypy supports annotations for class and instance 
+variables in class bodies and methods. Use ``ClassVar`` to
+indicate to the static type checker that this variable 
+should not be set on instances. 
```

Comment:
Maybe contrast this to a class attribute/variable without a ``ClassVar`` annotation: it can used as a class variable, but mypy won't prevent it from being used as an instance variable as well, potentially leading to ambiguity.